### PR TITLE
Sa 2496 admu exe sign

### PR DESCRIPTION
**Pull request by jworkmanjc**
## Issues
* [SA-2496](https://jumpcloud.atlassian.net/browse/SA-2496) - ADMU Replace Signing Cert

## What does this solve?

Our previous security software signing certificate vendor GoDaddy issued a notice this year that they would no longer be renewing signing certificates. This release addresses this issue by replacing our software signing cert with a DigiCert certificate. 

The signing process was moved from a hosted location to a GitHub Secrets/ Actions workflow. 

## Is there anything particularly tricky?

See PR Comments

## Screenshots

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -14,59 +14,97 @@ build-and-sign-ADMU: &build-and-sign-ADMU
   - Pester Tests:
       requires:
         - Build
-  # - Sign Executable:
```
Comment (jworkmanjc) [inline .circleci/config.yml:17]: Sign EXE step taken out of build steps

File: .circleci/config.yml [hunk 2]
```diff
@@ -238,6 +265,36 @@ jobs:
       - store_artifacts:
           path: ./
           destination: artifact-file
+  Sign Exe:
```
Comment (jworkmanjc) [inline .circleci/config.yml:268]: Sign EXE kicks off the github dispatch action

File: .circleci/config.yml [hunk 3]
```diff
@@ -238,6 +265,36 @@ jobs:
...
+    steps:
+      - run:
+          name: Sign unsigned exe
+          command: |
+            if [ "${CIRCLE_PULL_REQUEST##*/}" != "" ];then
+              curl --request POST --url 'https://api.github.com/repos/TheJumpCloud/jumpcloud-ADMU/dispatches' --header "authorization: Bearer $GITHUB_TOKEN" --data '{"event_type": "Sign Exe"}' 
+            else
+              echo "Not a PR"
+            fi
+  draft-github-release:
```
Comment (jworkmanjc) [inline .circleci/config.yml:280]: This step creates a github draft release based on data from the changelog

File: .github/workflows/config.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+      - shell: pwsh
+        env:
+          BASE64_ENCODED_CERT: ${{ secrets.BASE64_ENCODED_CERT }}
+          CERTPASS: ${{ secrets.CERTPASS }}
+        run: |
+          echo Dispatch ran
+      # - name: Push changes
```
Comment (jworkmanjc) [inline .github/workflows/config.yml:13]: TODO: Uncomment these settings and add sign.ps1 steps here

**Review by kmaranionjc — APPROVED** [review]
Looks good

**Review by gweinjc — APPROVED** [review]

**Review by gweinjc — APPROVED** [review]

**Pull request merged**